shared/desktop/yarn-helper/action-creator-creator.js

```diff
@@ -25,11 +25,10 @@ type CompileActionFn = (ns: ActionNS, actionName: ActionName, desc: ActionDesc)
 
 function compile(ns: ActionNS, {prelude, actions}: FileDesc): string {
   return `// @flow
-/* eslint-disable */
-
 // NOTE: This file is GENERATED from json files in actions/json. Run 'yarn build-actions' to regenerate
+/* eslint-disable no-unused-vars,prettier/prettier */
 
-import {type PayloadType, type ReturnType} from '../constants/types/more'
+import {type ReturnType} from '../constants/types/more'
```

Comment:
`actions/waiting-gen` still imports `PayloadType` (maybe you fixed this in another part)

Reply:
yeah i changed it to import * as More